Doc/library/dbm.rst

```diff
@@ -33,6 +33,8 @@ the Oracle Berkeley DB.
    file's format can't be guessed; or a string containing the required module
    name, such as ``'dbm.ndbm'`` or ``'dbm.gnu'``.
 
+.. versionchanged:: 3.10
+   Accepts :term:`path-like object` for filename.
```

Comment:
Hum, not sure of the Python version.  Will it be released in 3.10 or 3.11?

Reply:
3.10 is a half step from release. Sorry, but it will be released in 3.11.

Reply:
No problem for me, just have to fix the doc.